runner_internal_test.go

```diff
@@ -82,26 +82,26 @@ func TestRunnerGroupTaskAndFlagArgs(t *testing.T) {
 			},
 		},
 		{
-			description: "Should exclude tasks passed directly to taskrunner",
-			cliArgs:     []string{"--config", "mock/task/1", "--longFlagA"},
+			description: "Should exclude flags and flag args passed directly to taskrunner",
+			cliArgs:     []string{"--config", "pathtoconfig", "--listAll", "--watch", "mock/task/1", "--longFlagA"},
 			expectedGroups: map[string][]string{
 				"mock/task/1": {"--longFlagA"},
 			},
 		},
 		{
-			// Validation for supported flags happens in the executor.
-			description: "Should include unsupported flag args in group",
-			cliArgs:     []string{"mock/task/1", "--longInvalidFlag"},
+			description: "Should exclude unsupported tasks or flags passed into group",
```

Comment:
nit: `should exclude unsupported tasks and flags passed to it`** to make it more clear we're not validating flags